mscore/musescore.cpp

```diff
@@ -2302,17 +2302,22 @@ static void mscoreMessageHandler(QtMsgType type, const QMessageLogContext &conte
      QByteArray localMsg = msg.toLocal8Bit();
 
      switch (type) {
+ #if (QT_VERSION >= QT_VERSION_CHECK(5, 5, 0))
```

Comment:
No way. If we put this every single time we use qInfo... the code will get cluttered. So or we swallow the pill or we don't, there is no in the middle solution.